packages/server/src/project/project.resolver.ts

```diff
@@ -26,4 +27,15 @@ export class ProjectResolver {
   updateProject(@Args('id') id: string, @Args('settings') settings: ConfigurableProjectSettings): Promise<ProjectModel> {
     return this.projectService.updateProject(id, settings);
   }
+
+  @Query(() => [UserModel])
+  projectUsers(@Args('projectId') projectId: string): Promise<UserModel[]> {
+    return this.projectService.getProjectUsers(projectId);
+  }
+
+  @ResolveField(() => [UserModel])
+  async users(@Parent() project: ProjectModel): Promise<UserModel[]> {
```

Comment:
```suggestion
  async users(@Parent() { id } : ProjectModel): Promise<UserModel[]> {
```
Could also do this since it's typed